_html/templates/form-work.php

```diff
@@ -0,0 +1,204 @@
+<?php 
+/** 
+* Sponsors template.
+*/
+
+include "inc/header.inc.php";
+include "modules/header.php";?>
+
+    <main class="clearfix" role="main">
```

Comment:
Why isn't all this mark-up in a different php file? Like it was done for all other templates? (a separate 'module')